tox.ini

```diff
@@ -12,6 +12,7 @@ deps =
     -ctests/constraints.txt
     -rtests/requirements-py3.txt
     # Extras
+    boto3>=1.13.0
```

Comment:
Is this really needed to test the changes in this pull request?

Reply:
Now it needs to perform `BatchDeliveriesTest::test_s3_export`. If we delete this line, the test will be skipped at every run.

Reply:
I think what @elacuesta means if whether or not `test_s3_export` could be reimplemented without adding a new dependency to the tests, which is a valid point. So, could it be reimplemented without boto3? And, if so, what would be the pros and cons of each test implementation?

Reply:
Using botocore (we already have this dependency) here means that we have to basically re-implement boto3 anyway, including authentication, all security protocols, exceptions. Also, we have to maintain it if S3 API changes.